include/node_base.h

```diff
@@ -17,6 +17,11 @@
 
 namespace mpm {
 
+namespace properties {
+//! Sclar Properties
+enum Scalar : unsigned int { Mass };
```

Comment:
Any thoughts of scoping the enum - like what we do in `particle` for solid, liquid and gas. I personally think there is no difference, but giving the scope might be cleaner.

Reply:
It's both particle and node properties.

Reply:
What I meant is to put like `Mass = 0, Something = 1` instead of just `Mass`

Reply:
It will automatically do that for you

Reply:
I agree, but then why we do it [here]?(https://github.com/cb-geo/mpm/blob/7d4fe7524cc4ebe58bd4593e86b69b8ed80d5639/include/particles/particle_base.h#L22)